src/ServiceControl.AcceptanceTests/TestSupport/ServiceControlComponentRunner.cs

```diff
@@ -45,7 +45,11 @@ public ServiceControlComponentRunner(ITransportIntegration transportToUse, Accep
 
         async Task InitializeServiceControl(ScenarioContext context)
         {
-            var settings = new Settings(instanceName, transportToUse.TypeName, persistenceToUse.PersistenceType, forwardErrorMessages: false, errorRetentionPeriod: TimeSpan.FromDays(10))
+            var logPath = Path.Combine(Path.GetTempPath(), Path.GetRandomFileName());
```

Comment:
I know you only moved things. This is a more general observation and not something that should hold up this PR. Generally, I wonder if we should put things into the folder that also contains other acceptance test outputs somewhere in the bin directory instead of spamming the temp drives.

Reply:
I'll look at this and potentially raise a follow-up PR.